lib/widgets/wrapper.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:vase/colors.dart';
+
+Widget themeWrapper({required Widget child}) {
```

Comment:
Can we make it a stateless widget instead of a functional widget please?

With class based widgets we gain performance benefits over functional widgets